digitalocean/loadbalancer/resource_loadbalancer.go

```diff
@@ -204,9 +204,10 @@ func resourceDigitalOceanLoadBalancerV0() *schema.Resource {
 			},
 
 			"forwarding_rule": {
-				Type:     schema.TypeSet,
-				Required: true,
-				MinItems: 1,
+				Type:          schema.TypeSet,
+				Optional:      true,
+				ConflictsWith: []string{"glb_settings"},
+				MinItems:      1,
```

Comment:
Forwarding rules aren't supported for GLB, instead `glb_settings` embed target rules as per requirement. Having `forwarding_rule` set as `required` is conflicting with the integration tests, hence changing this to optional. Should we update the resources readme text to reflect this change as well?

Reply:
Agreed 💯 taking note of it for later update once we GA.